File: Week 1/hello.c
#include <cs50.h>
#include <stdio.h>


int main(void){
    printf("hello,world! \n");
    string first = get_string("What's your first name? ");
    string last = get_string("What's your last name? ");
    printf("hello, %s %s\n",first,last);
    // printf("What's your name?");
    // char str[20];
    // fgets(str,20,stdin);
    // printf("hello, %s",str);

}

File: Week 6/hello.py
from cs50 import get_string


answer = get_string("what's your name?")
print("hello, " + answer)  # concactenation
print("hello,",answer) # it accepts and automatically gives you space between
print(f"hello, {answer}") # f stants for print format ,similar to printf in C
# print("hello, world")
